contracts/setup.js

```diff
@@ -0,0 +1,53 @@
+// Program Definition
+async function OP_SETUP (command = {}) {
+  // TODO: discuss w/ @ChronicSmoke
```

Comment:
@ChronicSmoke want you to take a look at this and any uses of the `Environment` class